07-HTML5 APIs/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html ng-app="app">
+<head>
+	<title>Topic 7 - HTML5 APIs</title>
+
+	<script src="node_modules/angular/angular.min.js"></script>
+	<script src="js/script.js"></script>
+	
+
+	<link rel="stylesheet" type="text/css" href="mystyle.css">
+
+</head>
+<body>
+
+	<header>
+		<h1>Topic 7</h1>
+	</header>
+
+	<section ng-controller="Ctrl">
+
+		<input type="text" name="fname" ng-model="item" required><br><br>
+
+		<button ng-model="button" ng-click="save()" id="save">Save</button>
+		<button ng-model="button" ng-click="clear()" id="clear">Clear</button>
+
+	</section><br><br>
+
+	<a href="canvas.html" target="_blank" id="canvasHTML"> >>Test Canvas html!! << </a> 
+
+	<p id="dnd">Drag and Drop</p>
+	<div id="drop_zone">Drop files here</div>
+	<output id="list"></output>
+
+
+<br><br><br>
```

Comment:
br's are not used as a styling tool, they just provoke a carriage-return. Try to use css to manages paddings and margins.